packages/reporter/cypress/support/utils.ts

```diff
@@ -18,131 +17,16 @@ interface HandlesFileOpeningProps {
-      cy.viewport(600, 600)
-
-      if (stackTrace) {
-        cy.contains('View stack trace').click()
-      }
-
-      cy.get(selector).first().click()
-    })
-
-    it('opens modal with available editors', () => {
-      _.each(availableEditors, ({ name }) => {
-        cy.contains(name)
-      })
-
-      cy.contains('Other')
-      cy.contains('Set preference and open file')
-    })
-
-    // NOTE: this fails because mobx doesn't make the editors observable, so
-    // the changes to the path don't bubble up correctly. this only happens
-    // in the Cypress test and not when running the actual app
-    it.skip('updates "Other" path when typed into', () => {
-      cy.contains('Other').find('input[type="text"]').type('/absolute/path/to/foo.js')
-      .should('have.value', '/absolute/path/to/foo.js')
-    })
-
-    describe('when editor is not selected', () => {
-      it('disables submit button', () => {
-        cy.contains('Set preference and open file')
-        .should('have.class', 'is-disabled')
-        .click()
-
-        cy.wrap(getRunner().emit).should('not.to.be.calledWith', 'set:user:editor')
-        cy.wrap(getRunner().emit).should('not.to.be.calledWith', 'open:file')
-      })
-
-      it('shows validation message when hovering over submit button', () => {
-        cy.get('.editor-picker-modal .submit').trigger('mouseover')
-        cy.get('.cy-tooltip').last().should('have.text', 'Please select a preference')
-      })
-    })
-
-    describe('when Other is selected but path is not entered', () => {
-      beforeEach(() => {
-        cy.contains('Other').click()
-      })
-
-      it('disables submit button', () => {
-        cy.contains('Set preference and open file')
-        .should('have.class', 'is-disabled')
-        .click()
-
-        cy.wrap(getRunner().emit).should('not.to.be.calledWith', 'set:user:editor')
-        cy.wrap(getRunner().emit).should('not.to.be.calledWith', 'open:file')
-      })
-
-      it('shows validation message when hovering over submit button', () => {
-        cy.get('.editor-picker-modal .submit').trigger('mouseover')
-        cy.get('.cy-tooltip').last().should('have.text', 'Please enter the path for the "Other" editor')
-      })
-    })
-
-    describe('when editor is set', () => {
-      beforeEach(() => {
-        cy.contains('Visual Studio Code').click()
-        cy.contains('Set preference and open file').click()
-      })
-
-      it('closes modal', function () {
-        cy.contains('Set preference and open file').should('not.exist')
-      })
-
-      it('emits set:user:editor', () => {
-        expect(getRunner().emit).to.be.calledWith('set:user:editor', availableEditors[4])
-      })
-
-      it('opens file in selected editor', () => {
-        expect(getRunner().emit).to.be.calledWith('open:file', {
-          where: availableEditors[4],
-          ...file,
-        })
+        expect(getRunner().emit).to.be.calledWith('open:file:unified')
```

Comment:
We continue to have token coverage here by asserting the correct event gets emitted, but the rest of the coverage will come from app/e2e.